polars/polars-core/src/series/mod.rs

```diff
@@ -949,6 +949,16 @@ impl Series {
         self.slice(-(len as i64), len)
     }
 
+    pub fn count_as_series(&self) -> Series {
```

Comment:
I don't think this is worth a series. A count is always of a know dtype and a `Series` exposes a `len` method. You can always wrap it yourself in a series.

And I also don't think that it ever should return null, every `Series` has a count.